### PR TITLE
ALBS-651: Merge alma-sbom type and format CLI parameters

### PR DESCRIPTION
**Pull request by soksanichenko**
- Type of SBOM record and a file format are merged into the one CLI argument
- Some refactoring (PEP8, code style)

### CODE REVIEW

File: alma_sbom.py [hunk 1]
```diff
@@ -49,6 +32,63 @@ class PackageNevra:
...
+    def __repr__(self):
+        if self.epoch is not None:
+            return f'{self.epoch}:{self.name}-' \
+                   f'{self.version}-{self.release}.{self.arch}'
+        else:
+            return f'{self.name}-{self.version}-' \
+                   f'{self.release}.{self.arch}'
+
+
+class UnsupportedFileFormatException:
```
Comment (javihernandez) [inline alma_sbom.py:44]: This is not being used. But, If you have plans to do so, I presume you want to create an exception, hence this class should inherit from `Exception` class, isn't it?

**Review by javihernandez — CHANGES_REQUESTED** [review]
Beautiful, thanks for this PR!
Left a minor comment, but also, can you update the instructions in README.md before merging?

File: alma_sbom.py (hunk 1 above)
Comment (soksanichenko) [reply alma_sbom.py:44]: I wanted to use them, but changed my mind and forgot to delete them =)

File: alma_sbom.py [hunk 2]
```diff
@@ -49,6 +32,59 @@ class PackageNevra:
...
+
+    supported_file_formats = defaultdict(list, **{
+        'cyclonedx': [
+            'json',
+            'xml',
+        ]
+    })
+
+    def __call__(self, sbom_type_file_format: str) -> FileFormat:
+        sbom_record_type, file_format = sbom_type_file_format.split(',')
```
Comment (javihernandez) [inline alma_sbom.py:69]: Sorry, I overlooked this. If memory serves me well (can't find the notes from that meeting), Eugene suggested to receive "cyclonedx-xml" or "cyclonedx-json", anything against doing it?

Comment (soksanichenko) [reply alma_sbom.py]: Nothing, done

**Review by javihernandez — APPROVED** [review]
LGTM.
Can you also get rid of `sbom-type` in the list of CLI arguments [here](https://github.com/AlmaLinux/alma-sbom/blob/main/README.md?plain=1#L28) before merging?

**Review by Korulag — APPROVED** [review]

**Pull request merged**